Classes/Hooks/Form/PrefillForEditUsageHook.php

```diff
@@ -222,28 +223,28 @@ protected function resolveExpressions(
     protected function getFileReference(int $eventUid, int $position): ?FileReference
     {
         $queryBuilder = $this->getQueryBuilderForTable('sys_file_reference');
-        $statement = $queryBuilder
+        $queryResult = $queryBuilder
             ->select('uid')
             ->where(
                 $queryBuilder->expr()->eq(
                     'tablenames',
-                    $queryBuilder->createNamedParameter('tx_events2_domain_model_event', \PDO::PARAM_STR)
+                    $queryBuilder->createNamedParameter('tx_events2_domain_model_event')
```

Comment:
[Connection::PARAM_STR] ???

Reply:
I know its the default one. But still its better to keep it for readability?
Its up to you ;)

Reply:
Nope. PARAM_STR is default. So no need to add that explizit.